drivers/eks/eks_driver.go

```diff
@@ -733,7 +734,68 @@ func getParameterValueFromOutput(key string, outputs []*cloudformation.Output) s
 func (d *Driver) Update(ctx context.Context, info *types.ClusterInfo, options *types.DriverOptions) (*types.ClusterInfo, error) {
 	logrus.Infof("Starting update")
 
-	// nothing can be updated so just return
+	state, err := getStateFromOptions(options)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing state: %v", err)
+	}
+
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(state.Region),
+		Credentials: credentials.NewStaticCredentials(
+			state.ClientID,
+			state.ClientSecret,
+			"",
+		),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error getting new aws session: %v", err)
+	}
+
+	svc := autoscaling.New(sess)
+
+	filterName := "key"
+	filterValue := "kubernetes.io/cluster/" + state.ClusterName
+
+	describeTagsResult, err := svc.DescribeTags(&autoscaling.DescribeTagsInput{
+		Filters: []*autoscaling.Filter{
+			&autoscaling.Filter{
+				Name:   &filterName,
+				Values: []*string{&filterValue},
+			},
+		},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error getting tags: %v", err)
+	}
+	if len(describeTagsResult.Tags) != 1 {
+		return nil, fmt.Errorf("found %d tags for update", len(describeTagsResult.Tags))
+	}
+
+	resourceID := describeTagsResult.Tags[0].ResourceId
+	describeAutoScalingGroupsResult, err := svc.DescribeAutoScalingGroups(&autoscaling.DescribeAutoScalingGroupsInput{
+		AutoScalingGroupNames: []*string{resourceID},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error getting auto scaling group: %v", err)
+	}
+	if len(describeAutoScalingGroupsResult.AutoScalingGroups) != 1 {
+		return nil, fmt.Errorf("found %d groups for update", len(describeAutoScalingGroupsResult.AutoScalingGroups))
+	}
+
+	autoScalingGroup := describeAutoScalingGroupsResult.AutoScalingGroups[0]
+	if (*autoScalingGroup.MinSize == state.MinimumASGSize) && (*autoScalingGroup.MaxSize == state.MaximumASGSize) {
```

Comment:
You should be able to tell the current size of the autoscaling group from the `cluster info`.  I would check if the size had changed based on that rather than making an expensive network call to AWS.  This could also be done immediately when the func is called rather than waiting to create an AWS client.